docs/imm-otel.md

```diff
@@ -0,0 +1,110 @@
+# Deploy the SignalFx Smart Agent and OpenTelemetry Collector for Infrastructure Monitoring
+
+SignalFx Infrastructure Monitoring helps you gather metrics related to your system's performance. Metrics provide information about processes running inside the system, including counters, cumulative counters, and gauges.
+
+The OpenTelemetry Collector is a stand-alone service that can ingest metrics from various sources. The OpenTelemetry collector provides additional flexibility in configuration options. You can now configure integrations to send data to an OpenTelemetry Collector to centrally manage data sent to SignalFx.
+
+This is two-step process:
+
+1. [Deploy the SignalFx Smart Agent](#deploy-the-signalfx-smart-agent)
+2. [Deploy an OpenTelemetry Collector for SignalFx](#deploy-an-opentelemetry-collector-for-signalfx)
+
+The following sections describe each step in detail.
+
+## Deploy the SignalFx Smart Agent
+
+The SignalFx platform is built to ingest, store, analyze, visualize, and alert on metrics at scale. A metric is anything that is measurable (you can assign a numerical value to it) and variable (changes over time). The following are examples of metrics:
+
+* CPU utilization % of a server
+* Response time in milliseconds of an API call
+* The number of unique users who logged in over the previous 24-hour period
```

Comment:
this may be unnecessary for the purpose of the topic.

instead perhaps we can talk about how the smart agent handles receiving metrics from integrations before exporting to the otel collector for central management and processing.